이찬균/index.css

```
@@ -0,0 +1,114 @@
+html,
+body {
+  background-color: #fff8f4;
+  color: #3f3f3f;
+}
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Do Hyeon", sans-serif;
+}

```

```suggestion
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  font-family: "Do Hyeon", sans-serif;
}
```

이렇게 한 줄 추가하면 조금 더 원하는 대로 CSS를 먹이기 쉽다는 점 기억해주세요 ~~

---

넵!